dipy/workflows/reconst.py

```diff
@@ -1032,3 +1033,198 @@ def get_fitted_ivim(self, data, mask, bval, bvec, b0_threshold=50):
+            out_peaks_values='peaks_values.nii.gz',
+            out_peaks_indices='peaks_indices.nii.gz', out_gfa='gfa.nii.gz'):
+        """Reconstruct the fiber local orientations using the Robust and
+        Unbiased Model-BAsed Spherical Deconvolution (RUMBA-SD) [1]_ model. The
+        fiber response function (FRF) is computed using the single-shell,
+        single-tissue model, and the voxel-wise fitting procedure is used for
+        RUMBA-SD.
+
+        Parameters
+        ----------
+        input_files : string
+            Path to the input volumes. This path may contain wildcards to
+            process multiple inputs at once.
+        bvalues_files : string
+            Path to the bvalues files. This path may contain wildcards to use
+            multiple bvalues files at once.
+        bvectors_files : string
+            Path to the bvectors files. This path may contain wildcards to use
+            multiple bvectors files at once.
+        mask_files : string
+            Path to the input masks. This path may contain wildcards to use
+            multiple masks at once.
+        b0_threshold : float, optional
+            Threshold used to find b0 volumes.
+        bvecs_tol : float, optional
+            Bvecs should be unit vectors.
+        roi_center : variable int, optional
+            Center of ROI in data. If center is None, it is assumed that it is
+            the center of the volume with shape `data.shape[:3]`.
+        roi_radii : int or array-like, optional
+            radii of cuboid ROI in voxels.
+        fa_thr : float, optional
+            FA threshold to compute the WM response function.
+        extract_pam_values : bool, optional
+            Save or not to save pam volumes as single nifti files.
+        sh_order : int, optional
+            Spherical harmonics order used in the CSA fit.
+        odf_to_sh_order : int, optional
+            Spherical harmonics order used for peak_from_model to compress
+            the ODF to spherical harmonics coefficients.
+        parallel : bool, optional
+            Whether to use parallelization in peak-finding during the
+            calibration procedure.
+        num_processes : int, optional
+            If `parallel` is True, the number of subprocesses to use
+            (default multiprocessing.cpu_count()). If < 0 the maximal number
+            of cores minus ``num_processes + 1`` is used (enter -1 to use as
+            many cores as possible). 0 raises an error.
+        gm_response : float, optional
+            Mean diffusivity for GM compartment. If `None`, then grey
+            matter volume fraction is not computed.
+        csf_response : float, optional
+            Mean diffusivity for CSF compartment. If `None`, then CSF
+            volume fraction is not computed.
+        n_iter : int, optional
+            Number of iterations for fODF estimation. Must be a positive int.
+        recon_type : {'smf', 'sos'}, optional
```

Comment:
`str,` instead of `{'smf', 'sos'}`. you can add this in the docstring

Reply:
Done in 8603b93.